src/l3.c

```diff
@@ -23,12 +23,19 @@
 
 #include "l3.h"
 
+#ifdef L3_LOC_ENABLED
+#include "loc.h"
+#endif  // L3_LOC_ENABLED
+
 /**
  * L3 Log entry Structure definitions:
  */
 struct l3_entry
 {
     pid_t       tid;
+#ifdef L3_LOC_ENABLED
+    loc_t       loc;
```

Comment:
Note that in a future PR, when this field is activated, we are -only- re-using the 4-byte hole that is conveniently left behind for us in current non-L3_LOC_ENABLED build:

```
(gdb) ptype/o struct l3_entry
/* offset      |    size */  type = struct l3_entry {
/*      0      |       4 */    pid_t tid;
/* XXX  4-byte hole      */                                <<< loc_t loc; field is using these pad bytes.
/*      8      |       8 */    const char *msg;
/*     16      |       8 */    uint64_t arg1;
/*     24      |       8 */    uint64_t arg2;

                               /* total size (bytes):   32 */
                             }
```